airbyte-integrations/connectors/source-mssql/metadata.yaml

```diff
@@ -9,7 +9,7 @@ data:
   connectorSubtype: database
   connectorType: source
   definitionId: b5ea17b1-f170-46dc-bc31-cc744ca984c1
-  dockerImageTag: 3.7.4
+  dockerImageTag: 3.7.6
```

Comment:
3.7.6 because .5 already exists in https://github.com/airbytehq/airbyte/pull/35739

Reply:
yeah, we're going to simplify this ridiculousness :)